liferay-tango/liferay-portal
# LPS-174632 Create EventSegmentCriteria

Motivation
=======
Create a new EventSegmentCriteriaContributor in order to support EventSegmentCriteria in DXP

[Link to story](https://issues.liferay.com/browse/LPS-171785)
[Link to task](https://issues.liferay.com/browse/LPS-174632)

Proposed Solution
========
Add new criteria type Analytics
Add a new EventSegmentCriteriaContributor of Analytics type that support a download-document-amd-media field.

TODO:
- Add integration tests (I can't add it because the EventSegmentCriteriaContributor is below a FF :()

Steps to Verify:
----------------
1. Activate FF feature.flag.LPS-171722
2. Try to Edit/Create a Segment.
1. Event Segment Criteria should appear in the right panel.

Screencaps:
----------------
![Captura de pantalla 2023-02-08 a las 10 50 46](https://user-images.githubusercontent.com/5776822/217509266-0925e7b5-4cac-4b3a-8f54-02c96c12be44.png)
